build.gradle

```diff
@@ -52,6 +52,7 @@ project.ext.externalDependency = [
   'codemodel': 'com.sun.codemodel:codemodel:2.2',
   'commonsCli': 'commons-cli:commons-cli:1.0',
   'commonsCodec': 'commons-codec:commons-codec:1.3',
+  'commonsCollections4': 'org.apache.commons:commons-collections4:4.0',
```

Comment:
We haven't traditionally cared about this, but can we copy the HashedMap implementation from Apache Commons if we need it? This is a 500 KB artifact that will be forced on consumers since pegasus:data is pulled in by anything that uses pegasus.

Reply:
Fixed